docs/installing.adoc

```diff
@@ -268,27 +268,33 @@ You may wish to review the link:docs/using.adoc[Using Atomic Developer
 Bundle] documentation before starting ADB, especially if you are using
 host-based tools.
 
-* Manually Downloading the Vagrant Box Image
+* Manually Downloading the Vagrant Box
 +
 Alternatively, you can manually download the vagrant box from
 http://cloud.centos.org/centos/7/atomic/images/[cloud.centos.org] using
 your web browser or curl. For example:
 +
 ....
-# To get the libvirt image
+# To get the libvirt box
 $ wget http://cloud.centos.org/centos/7/atomic/images/AtomicDeveloperBundle-<latest>.box
 
-# To get the virtual box image
+# To get the virtual box box
 $ wget http://cloud.centos.org/centos/7/atomic/images/AtomicDeveloperBundle-<latest>.box
 ....
 +
-Once you have downloaded the image, you can add it to `vagrant` with
+Once you have downloaded the box, you can add it to `vagrant` with
 this command:
 +
 ....
-# Add the image to vagrant
-$ vagrant box add adb <local path to the downloded image>
+# Add the box to vagrant
+$ vagrant box add <box-name> <local path to the downloaded box>
 ....
++
+After adding box, set an environment variable as:
++
+----
+export BOX=<box-name>
+----
```

Comment:
Add a line saying, this allows you to customize the box name as per your requirements.